log4j-core/src/main/java/org/apache/logging/log4j/core/config/plugins/convert/TypeConverters.java

```diff
@@ -224,11 +224,26 @@ public Double convert(final String s) {
     }
 
     /**
-     * Converts a {@link String} into a {@link Duration}.
+     * Converts a {@link String} into a {@link org.apache.logging.log4j.core.appender.rolling.action.Duration}.
      * @since 2.5
+     * @deprecated since 2.24.0 use {@link JavaDurationConverter}
      */
     @Plugin(name = "Duration", category = CATEGORY)
-    public static class DurationConverter implements TypeConverter<Duration> {
+    @Deprecated
+    public static class DurationConverter
+            implements TypeConverter<org.apache.logging.log4j.core.appender.rolling.action.Duration> {
+        @Override
+        public org.apache.logging.log4j.core.appender.rolling.action.Duration convert(final String s) {
+            return org.apache.logging.log4j.core.appender.rolling.action.Duration.parse(s);
+        }
+    }
+
+    /**
+     * Converts a {@link String} into a {@link Duration}.
+     * @since 2.24.0
+     */
+    @Plugin(name = "JavaDuration", category = CATEGORY)
+    public static class JavaDurationConverter implements TypeConverter<Duration> {
```

Comment:
Just checking: We don't have any usage of this yet, right?

Reply:
Nice catch!

For compatibility sake I had to revert the `age` field in the `IfLastModified` builder to "our" `Duration`. Therefore this one is not used.